source/common/protobuf/utility.h

```diff
@@ -68,6 +68,14 @@ class MessageUtil {
   static void loadFromFile(const std::string& path, Protobuf::Message& message);
 
   /**
+   * Convert between two protobufs via a JSON round-trip. This is used to translate arbitrary
```

Comment:
Question (mostly curious): Is this really the best way of doing this? Isn't there some internal way that the library can do this using reflection? Seems silly to have to go to JSON and back.

Reply:
I'm checking with some folks on this, but I think this is the most straightforward using the utilities available. I will add a TODO, it seems it should be possible to do something similar to the serialization utils but producing a Struct instead of string and vice versa.